content/en/Platform Deep Dive/Organization/Organization Settings/SAML SSO/_index.md

```diff
@@ -173,55 +197,74 @@ Once you've completed the setup, your application for Cobalt appears in the Goog
 
 If you create an application for Okta **manually**, use the following parameters.-->
 
-We recommend creating a non-gallery SAML application for Cobalt manually. For details, read Okta's [documentation](https://help.okta.com/en-us/Content/Topics/Apps/Apps_App_Integration_Wizard_SAML.htm?cshid=ext_Apps_App_Integration_Wizard-saml).
-
-| For This Parameter in Cobalt | Enter This Value from Okta |
-|---|---|
-| IdP SSO URL | Sign on URL (Identity Provider Single Sign-On URL) |
-| IdP Certificate | Signing Certificate (X.509 Certificate) |
+We recommend creating a **non-gallery SAML application** for Cobalt manually. For more information, refer to Okta [documentation](https://help.okta.com/en-us/Content/Topics/Apps/Apps_App_Integration_Wizard_SAML.htm?cshid=ext_Apps_App_Integration_Wizard-saml).
```

Comment:
I think we should not recommend, but tell them. The Cobalt gallery in Okta will NOT work with Auth0 until it's updated, which won't be for some time.